### PR TITLE
Readers: `atril`, `calibre`, `evince`

### PR DESCRIPTION
**Pull request by nobody43**

### CODE REVIEW

File: dists/flags/main.flags [hunk 1]
```diff
@@ -54,6 +55,8 @@ downloadhelper complain
 e2fsck complain
 etckeeper complain
 evince complain
```
Comment (nobody43) [inline dists/flags/main.flags:57]: Ready?

File: apparmor.d/groups/apps/calibre [hunk 2]
```diff
@@ -98,27 +142,30 @@ profile calibre @{exec_path} {
   owner /tmp/calibre_*_tmp_*/{,**} rw,
   owner /tmp/calibre-*/{,**} rw,
   owner /tmp/[0-9]*-*/ rw,
-  owner /tmp/[0-9]*-*/** rwl -> /tmp/[0-9]*-*/**,
+  owner /tmp/[0-9]*-*/** rwl,
+#  owner /tmp/[0-9]*-*/** rwl -> /tmp/[0-9]*-*/**,  # newer AA version
```
Comment (nobody43) [inline apparmor.d/groups/apps/calibre:146]: Or bug: https://lists.snapcraft.io/archives/apparmor/2019-October/012050.html

Don't know why it passes the test.

File: apparmor.d/groups/apps/calibre [hunk 3]
```diff
@@ -33,15 +33,59 @@ profile calibre @{exec_path} {
...
+  dbus send bus=session path=/org/gtk/vfs/mounttracker
+       interface=org.gtk.vfs.MountTracker
+       member=ListMountableInfo
+       peer=(name=:*),
+
   @{exec_path} mrix,
   /{usr/,}bin/python3.[0-9]* r,
 
-  /{usr/,}{s,}bin/ldconfig     rix,
+  /{usr/,}{s,}bin/ldconfig{,.real} rix,
```
Comment (nobody43) [inline apparmor.d/groups/apps/calibre:88]: Shouldn't `ldconfig` be an orphan profile/abstraction?

**Comment by roddhjav** [comment]
Thanks a lot. Merged

File: dists/flags/main.flags (hunk 1 above)
Comment (roddhjav) [reply dists/flags/main.flags:57]: Yes, I think it is ready.

File: apparmor.d/groups/apps/calibre (hunk 2 above)
Comment (roddhjav) [reply apparmor.d/groups/apps/calibre:146]: I will keep this in mind. If more recent AA version support it, they could be a system to change the profile accordingly during the build time.

File: apparmor.d/groups/apps/calibre (hunk 3 above)
Comment (roddhjav) [reply apparmor.d/groups/apps/calibre:88]: We keep it inside the calling profile as it varies a lot depending of the profile.

**Pull request merged**